src/platform/site-wide/announcements/components/ClaimIncreaseBanner.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import backendServices from '../../../../platform/user/profile/constants/backendServices';
+import brandConsolidation from '../../../brand-consolidation';
 
 export default function ClaimIncreaseBanner({ dismiss, isLoggedIn, profile }) {
-  if (profile.loading) return <div />;
+  if (brandConsolidation.isEnabled()) return null;
+
+  if (profile.loading) return null;
   if (
     isLoggedIn &&
     !profile.services.includes(backendServices.CLAIM_INCREASE_AVAILABLE)
   )
-    return <div />;
+    return null;
```

Comment:
I believe I started the trend of returning `<div/>` instead of `null`, but I think it makes more sense to return `null`.